src/components/ui/animated-border.tsx

```diff
@@ -28,7 +28,7 @@ export const AnimatedBorder: React.FC<AnimatedBorderProps> = ({
       />
       
       {/* Content */}
-      <div className="relative rounded-[inherit] bg-black" style={{ padding }}>
+      <div className="relative rounded-[inherit] bg-black h-full w-full" style={{ padding }}>
```

Comment:
Does this affect anything on the BluePrint home page?

Reply:
hmmm, yea I think I'll try to find a better fix using a custom prop - let me know if the updated version is acceptable